File: app/src/main/java/fiix/challenge/fiixexercise/javasample/Model/TriviaQuestion.java
package fiix.challenge.fiixexercise.javasample.Model;

public class TriviaQuestion {
    String question;
    String answer;

    public String getQuestion() {
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }

    public String getAnswer() {
        return answer;
    }

    public void setAnswer(String answer) {
        this.answer = answer;
    }

    public TriviaQuestion(String question, String answer) {
        this.question = question;
        this.answer = answer;
    }
}

File: app/src/main/java/fiix/challenge/fiixexercise/kotlinsample/db/model/TriviaQuestion.kt
package fiix.challenge.fiixexercise.kotlinsample.db.model

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "trivia")
data class TriviaQuestion(var question: String, var answer: String? = null){
    @PrimaryKey(autoGenerate = true)
    var id: Int = 0
    var answerRevealed: Boolean = false
}